command-line-interface.md

```diff
@@ -0,0 +1,260 @@
+# Runtime Command Line Interface
+
+This section defines the runtime command line interface version 1.0.0.
+It is one of potentially several [runtime APIs defined by this specification](README.md).
+
+## Versioning
+
+The command line interface is versioned with [SemVer v2.0.0][semver].
+The command line interface version is independent of the OCI Runtime Specification as a whole (which is tied to the [configuration format](config.md#specification-version)).
+For example, if a caller is compliant with version 1.1 of the command line interface, they are compatible with all runtimes that support any 1.1 or later release of the command line interface, but are not compatible with a runtime that supports 1.0 and not 1.1.
+
+## Global usage
+
+The [runtime][] MUST provide an executable (called `funC` in the following examples).
+That executable MUST support commands with the following template:
+
+```
+$ funC [global-options] <COMMAND> [command-specific-options] <command-specific-arguments>
+```
+
+## Global options
+
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST NOT start with hyphens.
+The option parsing MUST be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` (even for commands not specified in this document).
+If the runtime is invoked with an unrecognized command, it MUST exit with a nonzero exit code and MAY log a warning to stderr.
+Beyond the above rules, the behavior of the runtime in the presence of commands and options not specified in this document is unspecified.
+
+## Character encodings
+
+This API specification does not cover character encodings, but runtimes SHOULD conform to their native operating system.
+For example, POSIX systems define [`LANG` and related environment variables][posix-lang] for [declaring][posix-locale-encoding] [locale-specific character encodings][posix-encoding], so a runtime in an `en_US.UTF-8` locale SHOULD write its [state](#state) to stdout in [UTF-8][].
+
+## Commands
+
+### create
+
+[Create][create] a container from a [bundle directory][bundle].
+
+* *Arguments*
+    * *`<ID>`* Set the container ID to create.
+* *Options*
+    * *`--bundle <PATH>`* Override the path to the [bundle directory][bundle] (defaults to the current working directory).
```

Comment:
I'm not really convinced, my reading of that entire document is that you have to have an operation called `create` which must take the arguments for a `bundle` and a container ID. **However**, in the CLI the bundle path is implicitly the current directory so you don't need to explicitly pass it (but you can explicitly pass it if you want).

Maybe we should just fix the way `create` is explained but then you have to make a distinction about having no `bundle` means that the runtime can have an arbitrary default.

Ultimately it all boils down to the fact that using runC like this would get super annoying super quick:

```
% runc run -b . container
```